us/en/skymiles/blocks/columns/columns.css

```diff
@@ -25,6 +25,39 @@
     margin-top: 30px;
 }
 
+.columns.emphasized > div {
+    display: block;
+    width: 85%;
+    margin: 0 auto;
+}
+
+.columns.emphasized > div > div {
+    border:  1px solid var(--color-delta-gray-middle);
+    padding: 20px;
+    margin-bottom: 0;
+}
+
+.columns.emphasized > div > div:first-child {
+    text-align: left;
+}
+
+.columns.emphasized > div > div:last-child {
+    margin-bottom: 40px;
+    margin-left: 0;
+    text-align: left;
+}
+
+.columns.emphasized > div > div > p {
+    color: var(--color-delta-blue);
+    font-size: var(--heading-font-size-m);
+}
+
+.columns.emphasized > div > div > p:first-child {
```

Comment:
The selectors seem complex here. @davidnuescheler previously recommended me to keep selectors simple by absorbing some of the complexity to JS.

Reply:
I would say it's fine for now as this is a 1st iteration on this block variant. As we go through the rest of the pages we'll see how to refine this, and maybe in the end we just move this back to content formatting in Word and just keep plain layout stuff in the CSS

Reply:
Just to add, we try to focus more on including JS when we have a bunch of say <p> tags where you we need to style few of them. Considering that this column was pretty straight forward with just two divs within it, I tried to keep it simple with basic styling .But ofcourse if we see more variants for such block in other pages and DOM manipulation seems to be the need, we can surely go for it.